types/es-to-primitive/es5.d.ts

```diff
@@ -0,0 +1,5 @@
+declare function ToPrimitive(
+	input: any,
+	hint?: typeof String | typeof Number,
+): string | number | boolean | null | undefined;
```

Comment:
```suggestion
): string | number | bigint | boolean | null | undefined;
```
i think this can still produce a symbol, whether it’s the es5 algorithm or not.